fivex/api/format.py

```diff
@@ -275,14 +276,14 @@ def __post_init__(self):
         )
 
     @property
-    def pvalue(self):
-        if self.log_pvalue is None:
+    def log_pvalue(self):
```

Comment:
I must have missed this in the forwarded thread: new data format for the eqtl catalogue?

Reply:
The source data (from the full points data, which has been joined into the credible_sets data) contains unmodified p-values instead of -log10 p-values. As far as I can tell, that's how it's always been.

Reply:
> The source data (from the full points data, which has been joined into the credible_sets data) contains unmodified p-values instead of -log10 p-values. As far as I can tell, that's how it's always been.

Thanks, but in this case I'm very confused: why did the old code ever work?

Reply:
It's because you're using parse_pval_to_log to convert values read as nominal p-values into -log10 p-values for VariantContainer:
https://github.com/statgen/fivex/blob/a1ca7490e5fb18c92c7e83d4d4769da86f1d7b8c/fivex/api/format.py#L592
I simply neglected to do this for the corresponding CIContainer when I read in p-values:
https://github.com/statgen/fivex/blob/a1ca7490e5fb18c92c7e83d4d4769da86f1d7b8c/fivex/api/format.py#L522
I suppose it's simpler just to reuse parse_pval_to_log on line 522 instead of re-engineering the self property the way I did.

Reply:
So it sounds like two items in play:
1. Using a consistent way of parsing values would be excellent and is a valuable change, if time permits (I am on vacation and may lose track of this before I return; I mean, the whole point was to get all this confetti out of my brain)

2. It sounds like there was some copy-pasted code that was not used previously- is that correct? (eg, I am asking why this never worked, and the answer is, it was copy-pasted rather than written for a purpose)

Reply:
Yes, (1) is good.
(2) The previous code (for parsing VariantContainer, for reading in raw points data), which includes parse_pval_to_log, was used, and does work. The actual problem is that I did not copy-and-paste the same parsing code to parse CIContainer, which, because of the update, now contains extra columns from the raw points data. If I copied the code directly, I would've noticed the extra parsing of p-values, but I simply wrote extra variables to hold the extra data. (P-values in the region view table are new to this update, so there was no previous version that didn't work correctly)